about/index.md

```diff
@@ -1,97 +1,86 @@
-independently of each other. Community contributors can also develop add-on
-modules to offer unique capabilities. Forseti's core modules work together to
-provide their respective features, and provide a foundation that addons can
-build upon.
+Forseti Security is a collection of community-driven, open-source tools to help you improve the
+security of your Google Cloud Platform (GCP) environments. Forseti consists of core modules that
+you can enable, configure, and execute independently of each other. Community contributors are also
+developingadd-on modules to offer unique capabilities. Forseti’s core modules work together, and
+provide a foundation that others can build upon.
 
 Get started with
 [Forseti Security]({% link _docs/latest/configure/configuring-forseti.md %}).
 
 ## When to use Forseti Security
 
-Use Forseti Security when you want to monitor your GCP resources to ensure that
-role-based access controls are set as you intended. Forseti Security can notify
-you if policies change unexpectedly and work automatically to keep your access
-policies in a known state. By using Forseti Security's modules together, you can
-make sure that there isn't any unauthorized access to your GCP resources.
+Forseti Security makes sense when you need Security at Scale. It’s easy to monitor a few resources
+in one or two projects, but at some point manual checking does not work anymore. If you want to
+systematically monitor your GCP resources to ensure that access controls are set as you intended,
+Forseti will allow creating rule-based policies to codify your security stance. Then, if something
+changes unexpectedly, action will be taken including notifying you, and possibly automatically
+reverting to a known state. 
+
+Taken as a whole, Forseti allows you to ensure your security is governed by consistent,
+intelligible rules.
 
 ## How Forseti Security works
 
-When you install Forseti Security, you can configure and deploy core Forseti
-Security modules. The core Forseti Security modules work together to take a
-snapshot of your GCP resources, monitor those resources, and notify you of
-access policy changes.
+When you install Forseti Security, you deploy the core Forseti Security modules and configure them
+to take a snapshot of your GCP resources, monitor those resources, and notify you of access policy
+changes.
 
 ### [Inventory]({% link _docs/latest/configure/inventory/index.md %})
 
-Inventory saves an hourly snapshot of your GCP resources to Cloud SQL, so you
-have a historical record of what was in your cloud. Using Inventory, you can
-understand all the resources you have in GCP and take action to conserve
-resources, reduce cost, and minimize security exposure. When configured,
-Inventory can run on a custom basis and send email notifications when it updates
-your resource snapshot.
+Inventory saves an inventory snapshot of your GCP resources to Cloud SQL, so you have a historical
```

Comment:
@blueandgold - We were going to call this 'crawler' rather then inventory, right? 

Crawler (puts data into) Inventory(snapshots), which then gets parsed by Scanner and actioned on by Notifier/Enforcer..

Reply:
Crawler is a very developer-centric name for a low-level module of the code. Inventory suffices here.

Reply:
OK. SG!